toplevel/vernac.ml

```diff
@@ -49,12 +49,17 @@ module State = struct
 
 end
 
-let emit_time state com tstart tend =
-  if state.State.time then Feedback.msg_notice (Topfmt.pr_cmd_header com ++ System.fmt_time_difference tstart tend)
+let emit_time state com tstart tend mstart mend =
+  if state.State.time then begin
+    let pp = Topfmt.pr_cmd_header com ++ System.fmt_time_difference tstart tend
+             ++ str " " ++ System.fmt_mem_difference mstart mend in
+    Feedback.msg_notice pp
+  end
 
 let interp_vernac ~check ~interactive ~state ({CAst.loc;_} as com) =
   let open State in
-    let tstart = System.get_time () in
+  let tstart = System.get_time () in
+  let mstart = Gc.quick_stat() in
```

Comment:
Is this code always performed? The reason I am asking is that `Gc.quick_stat` is a very slow function.  (And even slower with OCaml 5.) It is only named that way because it does not cause a major collection, contrarily to `Gc.stat`.

Reply:
yes
What does "very slow" mean here?

Reply:
With OCaml 5, several thousands of cycles, so a few microseconds. Nothing dramatic, but it adds up.

Reply:
It's only once per command so maybe not enough to worry about.